docs/example_go.md

```diff
@@ -3,29 +3,273 @@ title: "Example: Go"
 layout: docs
 ---
 
-You're a Go developer. You want to set up Tilt with your project.
+The best indicator of a healthy development workflow is a short feedback loop.
 
-We here you! We're working on a better guide for this.
+Kubernetes is a huge wrench in the works.
 
-For now, check out:
+Let's fix this.
 
-- [abc123](https://github.com/windmilleng/abc123) a mini microservice app with a Go server called `fe`.
-- The [Live Update Tutorial](live_update_tutorial.html), which optimizes a Go server.
-- Servantes (below), our microservice app in multiple languages.
+In this example, we're going to take you through a very simple shell script that
+serves static HTML.
+
+We'll use Tilt to:
+
+- Run the server on Kubernetes
+- Measure the time from a code change to a new process
+- Optimize that time for fast feedback
+
+Obviously, this is a silly example. But it can be a useful example to confirm that Tilt is working
+as expected in your environment.
+
+All the code is in this repo:
+
+[tilt-example-go](https://github.com/windmilleng/tilt-example-go)
+
+To skip straight to the fully optimized setup, go to this subdirectory:
+
+[Recommended Tiltfile](https://github.com/windmilleng/tilt-example-go/blob/master/3-recommended/Tiltfile)
+
+## Step 0: The Simplest Deployment
 
-## Servantes
+Our server is a simple HTML server that uses Gorilla Mux for routing requests:
```

Comment:
The package name is usually written as `gorilla/mux`; I couldn't find any instance of "Gorilla Mux." (https://github.com/gorilla/mux)